multimodal-search-tirg/README.MD

```diff
@@ -0,0 +1,7 @@
+# Multimodal Search With TIRG
+
+### Get model  
+wget https://storage.googleapis.com/image_retrieval_css/pretrained_models/checkpoint_fashion200k.pth  
```

Comment:
maybe we can indicate in the title this is the model for tirg for fanshion 200k

Reply:
We will iterate on this once we decide which one we use